ssl/ssl_conf.c

```diff
@@ -520,6 +520,22 @@ static int cmd_DHParameters(SSL_CONF_CTX *cctx, const char *value)
     return rv > 0;
 }
 #endif
+
+static int cmd_RecordPadding(SSL_CONF_CTX *cctx, const char *value)
+{
+    size_t block_size = (size_t)atoi(value);
+
+    if (block_size > 1 &&
+        block_size < SSL3_RT_MAX_PLAIN_LENGTH) {
```

Comment:
Probably it is reasonable to let people set a block_size of SSL3_RT_MAX_PLAIN_LENGTH (i.e. should be <= not <). In practice the amount of padding added would never be exactly SSL3_RT_MAX_PLAIN_LENGTH because we never send empty records - but still - I think we should allow people to configure that value.